app/components/agenda/delete.js

```diff
@@ -12,11 +12,10 @@ export default class AgendaDeleteComponent extends Component {
     this.isShowingWarning=!this.isShowingWarning;
   }
 
-  @action
-  async delete(){
-    await this.args.agendapunt.destroyRecord();
-    this.args.toggleEditing();
-    this.args.afterSave();
-  }
+  // @action
```

Comment:
commented out code should just be removed, if there is a reason to keep it it should be included in the comment block